basic/README.md

```diff
@@ -57,13 +57,15 @@ mvn clean -Dconnector=http-remoting -Dcontext.type=wildfly-naming-client -Dauthe
 Deploy jboss-client.jar from server distribution as dependency to local maven repo
 
 ```
+cd basic
 mvn -f pom-wildfly-client.xml clean install -Dprepare -Dserver.zip.url=url/to/zip/distribution/of/server.zip
 ```
 
 Examples:
 ```
+cd basic
 mvn -f pom-wildfly-client.xml clean install -Dprepare -Dserver.zip.url=file:///path/to/wildfly-28.0.0.Beta1.zip
-mvn -f pom-wildfly-client.xml clean install -Dprepare -Dserver.zip.url=https://github.com/wildfly/wildfly/releases/download/28.0.0.Beta1/wildfly-28.0.0.Beta1.zip
+mvn -f pom-wildfly-client.xml clean install -Dprepare -Dserver.zip.url=https://github.com/wildfly/wildfly/releases/download/30.0.1.Final/wildfly-30.0.1.Final.zip
```

Comment:
when updating this to WF 30.0.1, you could update other references to WF version too

Reply:
+1, WF version updated on more places